data/mods/BombasticPerks/perkmenu.json

```diff
@@ -51,61 +51,17 @@
       },
       {
         "condition": {
-          "and": [
-            {
-              "not": {
-                "or": [
-                  { "u_has_trait": "perk_way_openpalm" },
-                  { "u_has_trait": "perk_way_closedfist" },
-                  { "u_has_trait": "perk_way_pinchedfingers" }
-                ]
-              }
-            },
-            { "u_has_skill": { "skill": "unarmed", "level": 2 } },
-            { "u_has_skill": { "skill": "cutting", "level": 1 } }
-          ]
+          "not": {
+            "or": [
+              { "u_has_trait": "perk_way_openpalm" },
+              { "u_has_trait": "perk_way_closedfist" },
+              { "u_has_trait": "perk_way_pinchedfingers" }
+            ]
+          }
         },
-        "text": "Gain [<trait_name:perk_way_openpalm>]",
+        "text": "Gain [Way Of The Fist]",
         "topic": "TALK_PERK_MENU_OPENPALM"
```

Comment:
Shouldn't this topic be renamed if it leads to more than just Way of the Open Palm 🤔

Reply:
probably, yeah